classes/class-woothemes-sensei-certificates.php

```diff
@@ -1084,43 +1084,28 @@ public function certificate_link( $message, $course_id = 0, $user_id = 0 ) {
 		$view_link_profile  = Sensei()->settings->settings['certificates_view_profile'];
 		$is_viewable        = false;
 
-		if (
-			(
-				(
-					is_page( $my_account_page_id )
-					|| is_singular( 'course' )
-					|| isset( $wp_query->query_vars['course_results'] )
-				)
-				&& $view_link_courses
-			) || (
-				isset( $wp_query->query_vars['learner_profile'] )
-				&& $view_link_profile
-			)
-		) {
+		if ( ( 'page' == get_post_type( $my_account_page_id )
```

Comment:
Maybe leave a comment about the difference between these checks?

Reply:
Updated in https://github.com/woocommerce/sensei-certificates/pull/348/commits/0f6b5b7a18b3bd7739e00e03984e6ba99162b69c.

Reply:
I decided not to put that in since a Google search will give you the answer. 🙂